src/libexpr-c/nix_api_value.cc

```diff
@@ -15,9 +15,9 @@
 #include "value/context.hh"
 
 #ifdef HAVE_BOEHMGC
-# include "gc/gc.h"
-# define GC_INCLUDE_NEW 1
-# include "gc_cpp.h"
+#include "gc/gc.h"
+#define GC_INCLUDE_NEW 1
+#include "gc_cpp.h"
```

Comment:
Please don't do this, the indentation between `ifdef` and `endif` helps legibility.

Reply:
Did `clang-format` with our settings do this by any chance?

Reply:
If so, https://github.com/NixOS/nix/pull/10493 will ensure it won't anymore.

Reply:
yep, that was caused by `cland-format` settings (my editor auto-formats it on save)
I'm going to revert it